kubernetes/chart/vault-ui/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: vault-ui
+version: 0.1.0
```

Comment:
Can we align the version and description here with the current vault-ui version number and summary?